pkg/securityhubcollector/security_hub_collector_test.go

```diff
@@ -139,118 +23,250 @@ func outputEqual(a, b [][]string) bool {
-			"AwsEc2Vpc",
-			"Suppressed Test Finding Title",
-			"Suppressed Test Finding",
-			"HIGH",
-			"Do the thing",
-			"https://example.com/dothething",
-			"arn:aws:ec2:us-test-1:000000000001:vpc/vpc-00000000000000001",
-			"000000000001",
-			"FAILED",
-			"ACTIVE",
-			"SUPPRESSED",
-			"2020-03-22T13:22:13.933Z",
-			"2020-03-22T13:22:13.933Z",
+			name: "Suppressed finding",
+			finding: types.AwsSecurityFinding{
+				AwsAccountId: aws.String("000000000001"),
+				CreatedAt:    aws.String("2020-03-22T13:22:13.933Z"),
+				Description:  aws.String("Suppressed Test Finding"),
+				ProductArn:   aws.String("arn:aws:securityhub:us-east-1::product/aws/securityhub"),
+				RecordState:  types.RecordStateActive,
+				Resources: []types.Resource{
+					{
+						Id:     aws.String("arn:aws:ec2:us-test-1:000000000001:vpc/vpc-00000000000000001"),
+						Type:   aws.String("AwsEc2Vpc"),
+						Region: aws.String("us-east-1"),
+					},
+				},
+				SchemaVersion: aws.String("2018-10-08"),
+				Title:         aws.String("Suppressed Test Finding Title"),
+				UpdatedAt:     aws.String("2020-03-22T13:22:13.933Z"),
+				Workflow:      &types.Workflow{Status: types.WorkflowStatusSuppressed},
+				Severity:      &types.Severity{Label: types.SeverityLabelHigh},
+				Remediation: &types.Remediation{
+					Recommendation: &types.Recommendation{
+						Text: aws.String("Do the thing"),
+						Url:  aws.String("https://example.com/dothething"),
+					},
+				},
+				Compliance: &types.Compliance{Status: types.ComplianceStatusFailed},
+			},
+			expected: [][]string{
+				{
+					"Test Team 1",
+					"AwsEc2Vpc",
+					"Suppressed Test Finding Title",
+					"Suppressed Test Finding",
+					"HIGH",
+					"Do the thing",
+					"https://example.com/dothething",
+					"arn:aws:ec2:us-test-1:000000000001:vpc/vpc-00000000000000001",
+					"000000000001",
+					"us-east-1",
+					"dev",
+					"FAILED",
+					"ACTIVE",
+					"SUPPRESSED",
+					"2020-03-22T13:22:13.933Z",
+					"2020-03-22T13:22:13.933Z",
+				},
+			},
 		},
 	}
 
 	h := HubCollector{}
 	teamName := "Test Team 1"
+	environment := "dev"
 
-	if !outputEqual(h.convertFindingToRows(activeSecurityFinding, teamName), activeExpect) {
-		t.Errorf("ERROR: Active finding conversion does not match expectations")
-	}
-
-	if !outputEqual(h.convertFindingToRows(multiResourceSecurityFinding, teamName), multiResourceExpect) {
-		t.Errorf("ERROR: MultiResource finding conversion does not match expectations")
-	}
-
-	if !outputEqual(h.convertFindingToRows(noComplianceSecurityFinding, teamName), noComplianceExpect) {
-		t.Errorf("ERROR: NoCompliance finding conversion does not match expectations")
-	}
-
-	if !outputEqual(h.convertFindingToRows(suppressedSecurityFinding, teamName), suppressedExpect) {
-		t.Errorf("ERROR: Suppressed finding conversion does not match expectations")
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			actual := h.convertFindingToRows(tc.finding, teamName, environment)
```

Comment:
again, not new to this change, but things that factor into the test output (`teamName`, `environment`) should probably be part of the test input, no? or at the very least they should be defined as variables at the package level and referenced in the test output